tests/integration/config/plugins/integration-tests/data/disabling troublesome missions.txt

```diff
@@ -0,0 +1,6 @@
+disable mission "Conversation on Sayaiban"
+disable mission "Drak guarding Korath space"
+disable mission "Drak guarding Sayaiban"
+disable mission "Drak guarding Void Sprites"
+disable mission "Relic of the Past"
+disable mission "Stationed Augen"
```

Comment:
We can do this?? We should also disable landing on ruin mission then I had to set a condition for it being offered everywhere
Could be in another PR though